package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "zadanie-7.6.1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run test:html",
+    "test:html": "glob-run nu-html-checker *.html",
+    "build": "npm run build:clean && npm run build:sass && npm run build:autoprefixer && npm run test",
+    "build:sass": "sass --sourcemap=none --update sass:css",
+    "build:autoprefixer": "autoprefixer-cli css/style.css",
+    "build:clean": "rm -rf",
```

Comment:
where is path to files you want remove? try to install and use `RimRaf` instead of `rm -rf`